pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -27,29 +27,41 @@
     <PrereleaseLibraryPackage Include="System.Configuration.ConfigurationManager" />
     <PrereleaseLibraryPackage Include="System.Data.Odbc" />
     <PrereleaseLibraryPackage Include="System.Data.DataSetExtensions" />
-    <PrereleaseLibraryPackage Include="System.Data.SqlClient" />
-    <PrereleaseLibraryPackage Include="System.Drawing.Common" />
+    <PrereleaseLibraryPackage Include="System.Data.SqlClient">
+      <Version>4.6.0</Version>
+    </PrereleaseLibraryPackage>
+    <PrereleaseLibraryPackage Include="System.Drawing.Common">
```

Comment:
Can you explain the changes for Drawing, Pipes etc?

I was expecting to only see SqlClient -- if I diff v2.1 and v2.2, the only product changes are to SqlClient, Tracing, Compression, and Http - and the last 3 are OOB and not in the compat pack.

Reply:
This was per our offline discussion where we need to keep the latest compat pack up to date and react to changes happening on 2.1 as well. We agreed that we're going to service the compat pack from the 2.2 branch rather than from both to always have the latest one point to the right versions of packages.